src/poliastro/czml/utils.py

```diff
@@ -35,18 +36,21 @@ def ellipsoidal_to_cartesian(a, b, lat, lon):
 def intersection_ellipsoid_line(x, y, z, u1, u2, u3, a, b, c):
     """
     Intersection of an ellipsoid defined by its axises a, b, c with the
-    line p + λu
+    line p + λu.
+
     Parameters
     ----------
-    x, y, z: float
+    float x, y, z
```

Comment:
Interesting, I didn't know this syntax. Could you point out to the reST, docutils, Sphinx or napoleon documentation where you learned it?

Reply:
Actually I figured it out all by myself :sweat_smile: